launch/openni2.launch

```diff
@@ -100,6 +103,8 @@
       <arg name="rgb"                             value="$(arg rgb)" />
       <arg name="ir"                              value="$(arg ir)" />
       <arg name="depth"                           value="$(arg depth)" />
+      <arg name="depth_registered"                value="$(arg depth_registered)" />
+      <arg name="depth_registered_filtered"       value="$(arg depth_registered_filtered)" />
```

Comment:
In [processing.launch](https://github.com/ros-drivers/openni2_launch/blob/55c6d06be95d063220b20e35d007f97bec231d3e/launch/includes/processing.launch) I don't see `depth_registered_/filtered` args. Has this change been tested?

Reply:
My bad, I was referring to a wrong file. Indeed I do find these args in [rgbd_launch/launch/includes/processing.launch.xml](https://github.com/ros-drivers/rgbd_launch/blob/2a29e78fab15a63fd1cefdb4e0166f08004bf8d4/launch/includes/processing.launch.xml#L28).